### PR TITLE
Adding first input to global index pages documentation

### PR DESCRIPTION
**Pull request by MayaBerd**

### CODE REVIEW

File: docs/user-guide/home/global-modules/README.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  title: Global modules
+  priority: 999
+description: Global modules in OpenProject
+keywords: global modules, project overarching modules, global index pages
+---
+# Global modules
+
+Global modules present an overview of the projects that you are a member of or have permissions to see. Here you will find a summary of all the entries in the respective modules across your projects, such as **Activity, Work packages, Calendars, Boards**, etc.
+
+To find **Global modules** menu simply click on the logo in the header of the application, it will be displayed on the left side.
```
Comment (marcalcobe) [inline docs/user-guide/home/global-modules/README.md]: @MayaBerd maybe we should add also that you can access the global modules from the top right grid menu icon.

File: docs/user-guide/home/global-modules/README.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+**Projects** global module will display a list of the projects that exist in your OpenProject instance, which you are a member of and/or have the right to see (for example as an administrator) and public projects.
+
+You can create a new project here directly by using the  **+ Project** button at the bottom of the left side menu.
+
+![OpenProject projects overview in the global modules menu](openproject_global_modules_projects.png)
+
+On the left you will have the following options:
+
+**All projects** is the default view and shows only active projects. If you do not have administrator rights, this view will be the same as **My projects** for you.
```
Comment (marcalcobe) [inline docs/user-guide/home/global-modules/README.md]: The part of administration rights and being the same as My projects is not true. This menu entry will contain both projects that you are member of and Public projects independent of your rights on them.

File: docs/user-guide/home/global-modules/README.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+**My projects** is the view that will show projects you are a member of and public projects. When this menu entry is selected, an **I am member** filter will be activated under Filters. You can then add various other filters to adjust the view.
```
Comment (marcalcobe) [inline docs/user-guide/home/global-modules/README.md]: I don't think this is correct. In the My projects section the users will only see the projects where they are members, not the public projects.

File: docs/user-guide/home/global-modules/README.md (hunk 2 above)
Comment (marcalcobe) [reply docs/user-guide/home/global-modules/README.md]: It is true though that only active projects are displayed and archived projects are not.

File: docs/user-guide/home/global-modules/README.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+>Note: **Changesets** filter comes from repositories that are managed by OpenProject. For example, if you make a commit to a GIT or SVN repository, these changes will be displayed here. At the moment this filter is only relevant for self-hosted editions.
+
+
+
+## Work packages
+
+**Work packages** global module will show a list of all the work packages from the projects you are a member of and all public projects. You can select your **Favorite** and **Default** work package filters in the left side menu.
+
+PUBLIC - ???
```
Comment (marcalcobe) [inline docs/user-guide/home/global-modules/README.md]: The work package module work the same as inside a project. You could have work package table views marked as Public or Private. Note that if a view is marked as Private or Public but also Favourite they will be shown in the Favourite section.

File: docs/user-guide/home/global-modules/README.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+You can adjust the view by selecting respective filters on the left and clicking the **Apply** button. 
+
+>Note: **Changesets** filter comes from repositories that are managed by OpenProject. For example, if you make a commit to a GIT or SVN repository, these changes will be displayed here. At the moment this filter is only relevant for self-hosted editions.
+
+
+
+## Work packages
+
+**Work packages** global module will show a list of all the work packages from the projects you are a member of and all public projects. You can select your **Favorite** and **Default** work package filters in the left side menu.
```
Comment (marcalcobe) [inline docs/user-guide/home/global-modules/README.md]: I think we should change list for table.

File: docs/user-guide/home/global-modules/README.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+![openproject_user_guide_add_new_board](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_add_board.png)
+
+Here you can name the board, select a project to which a board should be assigned to and choose the board type. Find out more about editing boards [here](../../agile-boards). 
+
+## News
+
+**News** global module will display all news that have been published in projects you are a member of, have administrative privileges to see and public projects. 
+
+TIMEFRAME - ?
```
Comment (marcalcobe) [inline docs/user-guide/home/global-modules/README.md]: As far as I know there is no timeframe for news, it is a paginated view until the very first new created.

File: docs/user-guide/home/global-modules/README.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+
+**Time and costs** global module will list time and cost reports created by you and the ones set to be public in the left hand menu. The filters you see initially will be pre-filled based on the values you used for filtering during your last session. You can create new time and cost reports by adjusting filters, grouping criteria and units accordingly and clicking **Apply** button. Do not forget to **Save** your report. 
+
+![openproject_user_guide_global_time_cost](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_time_costs.png)
+
+Read more about creating and editing **Time and cost reports** [here](../../time-and-costs/reporting/).
+
+## Meetings
+
+**Meetings** global module will provide a list of all upcoming an past meetings you have created, been invited to, attended or have administrative privileges to see. The default view will show the **Upcoming meetings**. The most current meetings are shown on top of the list.
```
Comment (marcalcobe) [inline docs/user-guide/home/global-modules/README.md]: I don't think "the most current meetings" is clear, maybe "the closest meetings"?

File: docs/user-guide/home/global-modules/README.md [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+
+## Meetings
+
+**Meetings** global module will provide a list of all upcoming an past meetings you have created, been invited to, attended or have administrative privileges to see. The default view will show the **Upcoming meetings**. The most current meetings are shown on top of the list. 
+
+![openproject_user_guide_global_meetings](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_meetings.png)
+
+You can also choose to see all **Past meetings** or apply one of the filters based on your **Involvement** in a meeting. 
+
+You can create a new meeting directly from within the global modules menu by clicking a green ***Meeting** button either in the top right corner at the bottom of the sidebar.
```
Comment (marcalcobe) [inline docs/user-guide/home/global-modules/README.md]: "+Meeting" instead of "*Meeting"

**Review by marcalcobe — COMMENTED** [review]
I've added some comments with my feedback.

**Review by psatyal — COMMENTED** [review]
I made some changes to in a later commit (mostly phrasing). Most of @marcalcobe's comments seem to still apply, however. I've resolved the ones I address in my commit :)

File: docs/user-guide/home/global-modules/README.md (hunk 7 above)
Comment (psatyal) [reply docs/user-guide/home/global-modules/README.md]: I changed this in my commit to: "The default view will show the **Upcoming meetings**, in chronological order (earliest first)."

File: docs/user-guide/home/global-modules/README.md (hunk 8 above)
Comment (psatyal) [reply docs/user-guide/home/global-modules/README.md]: Fixed in my commit.

File: docs/user-guide/home/global-modules/README.md (hunk 1 above)
Comment (MayaBerd) [reply docs/user-guide/home/global-modules/README.md]: Excellent suggestion, I added it. Thank you.

File: docs/user-guide/home/global-modules/README.md (hunk 3 above)
Comment (MayaBerd) [reply docs/user-guide/home/global-modules/README.md]: yes, you are right. removed the part on public projects.

File: docs/user-guide/home/global-modules/README.md (hunk 4 above)
Comment (MayaBerd) [reply docs/user-guide/home/global-modules/README.md]: adding this.

**Pull request merged**